website/addons/base/__init__.py

```diff
@@ -160,22 +160,17 @@ def _include_to_static(self, include):
 
     @property
     def icon(self):
-
         try:
             return self._icon
-        except:
+        except AttributeError:
             static_path = os.path.join('website', 'addons', self.short_name, 'static')
             static_files = glob.glob(os.path.join(static_path, 'comicon.*'))
-            image_files = [
+            return next((
```

Comment:
Okay, general idea of this is that the old version made a list and was expecting no more than one item and returns that item or None. So you've replaced it with a generator that either returns the one item or None. Yes?